pycbc/inference/sampler_base.py

```diff
@@ -379,6 +380,10 @@ def _write_samples_group(self, fp, samples_group, parameters, samples,
             Write results starting from the given iteration.
         end_iteration : {None, int}
             Write results up to the given iteration.
+        index_offset : int, optional
```

Comment:
Does this mean, if I have 10,000 iterations in an ``InferenceFile`` and I pass ``start_iteration=0``, ``end_iteration=5000``, and ``index_offset=5000`` then give me iterations 5000 through 10000?

``index_offset = file array indices - samples' indices`` isn't very clear to me.